src/sender.py

```diff
@@ -69,9 +69,10 @@ def consume_stream(self, id):
                 == self.serial_number
             ):
                 ip = stream_info["inputChannel"]["decoder"]["device"][
-                    "privateIpAddress"
+                    "publicIpAddress"
```

Comment:
I'm still getting 0:0:0:0:0:0:0:1 as the public ip when registering the device with 127.0.0.1 as the private ip
Am I missing something here?

Reply:
That's normal, the service will take the `publicIpAddress` from the http request headers. If you're running all the components on the same machine, then the service will see the incoming requests as coming from localhost. As you probably already konow, `127.0.01` is the IPv4 address for localhost and `0:0:0:0:0:0:0:1` is the IPv6 address for localhost.